cal/__init__.py

```diff
@@ -88,7 +88,9 @@ def users():
                         .filter(Event.start < now + dt.timedelta(weeks=1))
 
     users = {event.user for event in events}    # use set to make users unique
-    return jsonify(data=[user.to_json() for user in users])
+    users = [user.to_json() for user in sorted(list(users),  # sort by name
+                                               key=lambda user: user.name)]
```

Comment:
Could we do (one line, no `list(users)` call).
```Python
users = [user.to_json() for user in sorted(users, key=lambda user: user.name)]
```
If that's too long for pep8, we can do
```Python
users = [user.to_json() for user in sorted(users, key=lambda u: u.name)]
```